ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.runs.$runIndex.index.tsx

```diff
@@ -115,14 +115,13 @@ const RunComponent = () => {
         <Table>
           <TableHeader>
             <TableRow>
-              <TableHead>Run parameter</TableHead>
-              <TableHead>Value</TableHead>
+              <TableHead>Run details</TableHead>
               <TableHead className='sr-only'>Actions</TableHead>
             </TableRow>
           </TableHeader>
           <TableBody>
             <TableRow>
-              <TableCell>Created At</TableCell>
+              <TableCell>Created at</TableCell>
```

Comment:
Strictly speaking these (good) spelling changes here and below should go to a separate commit, as they are unrelated to the main topic of this commit.